platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -154,24 +147,6 @@ protected LDAPConnectionFactory createLdapConnectionFactory(String url, Boolean
             truststorePass = encryptService.decryptValue(truststorePass);
```

Comment:
Doesn't look like we need to be grabbing these SSL properties anymore if we aren't using them.